recordset.go

```diff
@@ -102,6 +103,24 @@ func (rcs *Recordset) IsActive() bool {
 	return rcs.active.Get()
 }
 
+var ErrRecordsetClosed = errors.New("aerospike: Recordset closed")
+
+// Read reads the next record from the Recordset. If the Recordset has been
+// closed, it returns ErrRecordsetClosed.
+func (rcs *Recordset) Read() (record *Record, err error) {
+	var ok bool
+	select {
+	case <-rcs.cancelled:
+	case record, ok = <-rcs.Records:
+	case err, ok = <-rcs.Errors:
```

Comment:
No problem. I've pulled this PR in the code, and changed it a bit. Thanks for your help and thoughts.